components/applications-service/integration_test/global_test.go

```diff
@@ -24,6 +24,13 @@ var (
 // NewHabServiceMsg returns a new hab service message with the provided args
 func NewHabServiceMsg(supID, app, env, group,
 	origin, name, version, release, health string) *applications.HabService {
+	return NewHabServiceChannelMsg(supID, app, env, group,
```

Comment:
I feel that this wrapper is going to make us follow a pattern we don't want to follow, that is creating multiple constructor functions `NewABC()`. IMO it would be worth changing this function to accept optional parameters and if any parameter is not provided, then take the defaults.

Something like:
```go
func NewHabServiceMsg(params map[string]string) *applications.HabService {
    // New message with defaults
    habServiceMsg := DefaultHabService()

    for key, value := range params {
        // Override here the provided parameters into 'habServiceMsg'
    }

    return habServiceMsg
}
```

Reply:
I feel like this obfuscates the parameters and makes it harder to use the method though.